docs/proxy/residential_proxy/nodejs_examples.md

```diff
@@ -92,51 +93,58 @@ Use a single IP from Germany for all requests done in the launched browser
         console.log(html);
     });
 
-## [](#usage-with-request) Usage with [request](https://www.npmjs.com/package/request) and [request-promise](https://www.npmjs.com/package/request-promise) NPM packages
+## [](#usage-with-request) Usage with our [requestAsBrowser](https://sdk.apify.com/docs/api/utils#utilsrequestasbrowseroptions)
```

Comment:
Hmm, now I see that we have multiple anchors with the same name. We need to differentiate them, otherwise they won't work.

Reply:
They will work because each of them has a different path right? e.g. `proxy/THIS-IS-DIFFERENT/nodejs-examples#usage-with-request`

Reply:
Ok, cool. Did not see this in the review.